ec2/sc-ec2-linux-jumpcloud.yaml

```diff
@@ -1,6 +1,11 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Description: 'Service Catalog: EC2 Reference Architecture(fdp-1oc5f3ula).'
 Metadata:
```

Comment:
would you mind moving these ignore checks to the specific resource we want to ignore?

Reply:
@zaro0508 Did this, it's a bit repetitive.